src/app/visual/vertical-tabset/vertical-tabs-visual.component.html

```diff
@@ -57,7 +59,7 @@ <h2>Vertical tabset</h2>
 
 <h2>Vertical tabset with no tab groups</h2>
 <div id="screenshot-vertical-tabs-no-groups" [@.disabled]="true">
-  <sky-vertical-tabset>
+  <sky-vertical-tabset [maintainTabContent]="maintainTabContent">
```

Comment:
Attributes should be on a new line (also see below for one more in this file).

Reply:
Done